CHANGELOG.md

```diff
@@ -2,12 +2,199 @@
+- The `WordPress.VIP.PostsPerPage` sniff has been split into two distinct sniffs:
+    - `WordPress.WP.PostsPerPage` which will check for the use of a high pagination limit and will throw a `warning` when this is encountered. For the `VIP` ruleset, the error level remains `error`.
+    - `WordPress.VIP.PostsPerPage` wich will check for disabling of pagination.
+- The default value for `minimum_supported_wp_version`, as used by a [number of sniffs detecting usage of deprecated WP features](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#minimum-wp-version-to-check-for-usage-of-deprecated-functions-classes-and-function-parameters), has been updated to `4.6`.
+- The `WordPress.WP.AlternativeFunctions` sniff will now only throw a warning if/when the recommended alternative function is available in the minimum supported WP version of a project.
+    In addition to this, certain alternatives are only valid alternatives in certain circumstances, like when the WP version only supports the first parameter of the PHP function it is trying to replace.
+    This will will now be taken into account for:
+    - `wp_strip_all_tags()` is only a valid alternative for the PHP native `strip_tags()` when the second parameter `$allowed_tags` has not been passed.
+    - `wp_parse_url()` only added support for the second parameter `$component` of the PHP native `parse_url()` function in WP 4.7.0.
+- The `WordPress.WP.DeprecatedFunctions` sniff will now detect functions deprecated in WP 4.9.
+- The `WordPress.WP.GlobalVariablesOverride` sniff will now display the name of the variable being overridden in the error message.
+- The `WordPress.WP.I18n` sniff now extends the `AbstractFunctionRestrictionSniff`.
+- Assignments in conditions in ternaries as detected by the `WordPress.CodeAnalysis.AssignmentInCondition` sniff will now be reported under a separate error code `FoundInTernaryCondition`.
+- The default error level for the notices from the `WordPress.DB.DirectDatabaseQuery` sniff has been lowered from `error` to `warning`. For the `VIP` ruleset, the error level remains `error`.
+- The default error level for the notices from the `WordPress.Security.PluginMenuSlug` sniff has been lowered from `error` to `warning`. For the `VIP` ruleset, the error level remains `error`.
+- The default error level for the notices from the `WordPress.WP.CronInterval` sniff has been lowered from `error` to `warning`. For the `VIP` ruleset, the error level remains `error`.
+- The `Sniff::get_function_call_parameters()` utility method now has improved handling of closures when passed as function call parameters.
+- Rulesets: a number of error codes were previously silenced by explicitely `exclude`-ing them. Now, they will be silenced by setting the `severity` to `0` which makes it more easily discoverable for maintainers of custom rulesets how to enable these error codes again.
```

Comment:
Typo: explicitely -> explicitly

Reply:
Why do they make English such a difficult language to spell ? ;-)

Reply:
Oddly enough, this is one of the words that always use to be a gotcha for me, which is why I now look out for it :-)

Reply:
*grin*, well, I think I have a number of those, though `(PHP)compatibility` is no longer amongst them ;-)